### PR TITLE
Post view restyling 

### PR DESCRIPTION
**Pull request by darya42185**
develop
https://github.com/ita-social-projects/dokazovi-fe/issues/264

### CODE REVIEW

File: src/lib/theme/typography.ts [hunk 1]
```diff
@@ -48,4 +48,16 @@ export const TYPOGRAPHY = {
     fontSize: '14px',
     lineHeight: '18px',
   },
+  postContent: {
+    fontFamily: 'Lato',
+    fontWeight: 700,
+    fontSize: '16px',
+    lineHeight: '26px',
+  },
+  dataCreatedAt: {
```
Comment (SergeOlabin) [inline src/lib/theme/typography.ts:57]:
```suggestion
  dateCreatedAt: {
```

File: src/modules/posts/styles/PostViewInfo.styles.ts [hunk 2]
```diff
@@ -2,6 +2,7 @@ import { makeStyles } from '@material-ui/core';
 
 export const useStyles = makeStyles({
   cardContainer: {
+    marginLeft: '45px',
     marginTop: '20px',
```
Comment (SergeOlabin) [inline src/modules/posts/styles/PostViewInfo.styles.ts:6]: pls use `theme.spacing()` here

File: src/modules/posts/styles/PostViewInfo.styles.ts [hunk 3]
```diff
@@ -37,12 +41,17 @@ export const useStyles = makeStyles({
...
   createdAt: {
-    position: 'absolute',
-    bottom: 0,
-    right: 0,
-    marginRight: '20px',
-    marginBottom: '20px',
+    color: '#3B6F95',
+  },
+  dataBox: {
+    margin: '20px',
```
Comment (SergeOlabin) [inline src/modules/posts/styles/PostViewInfo.styles.ts:54]: pls use theme.spacing() here

File: src/modules/posts/styles/PostViewInfo.styles.ts [hunk 4]
```diff
@@ -37,12 +41,17 @@ export const useStyles = makeStyles({
...
   },
   content: {
     margin: '20px',
```
Comment (SergeOlabin) [inline src/modules/posts/styles/PostViewInfo.styles.ts:57]: pls use theme.spacing() here

File: src/modules/posts/styles/PostViewInfo.styles.ts [hunk 5]
```diff
@@ -37,12 +41,17 @@ export const useStyles = makeStyles({
...
+    fontSize: '14px',
+    lineHeight: '24px',
+  },
   createdAt: {
-    position: 'absolute',
-    bottom: 0,
-    right: 0,
-    marginRight: '20px',
-    marginBottom: '20px',
+    color: '#3B6F95',
```
Comment (SergeOlabin) [inline src/modules/posts/styles/PostViewInfo.styles.ts:51]: pls use theme.palette here

File: src/modules/posts/styles/PostViewInfo.styles.ts [hunk 6]
```diff
@@ -37,12 +41,17 @@ export const useStyles = makeStyles({
     position: 'relative',
     minHeight: '550px',
   },
+  institution: {
+    fontFamily: 'Literata',
+    fontWeight: 200,
+    fontSize: '14px',
+    lineHeight: '24px',
```
Comment (SergeOlabin) [inline src/modules/posts/styles/PostViewInfo.styles.ts:48]: i think you should be able to use themming here

File: src/modules/posts/components/PostView.tsx [hunk 7]
```diff
@@ -20,63 +21,62 @@ const PostView: React.FC<IPostViewProps> = ({ post }) => {
...
-      <Card className={classes.cardContainer}>
-        <Box className={classes.authorBlock}>
-          <Box>
-            <CardMedia
-              style={{ padding: '15px', height: '58px', width: 46 }}
-              className={classes.avatar}
-              image={post?.author?.avatar}
-              title={authorFullName}
-            />
+    <ThemeProvider theme={MAIN_THEME}>
```
Comment (SergeOlabin) [inline src/modules/posts/components/PostView.tsx:24]: we have it provided in root , don't need here

File: public/index.html [hunk 8]
```diff
@@ -24,6 +24,13 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap" />
+    <link rel="preconnect" href="https://fonts.gstatic.com/%22%3E"/> 
+    <title>Dokazovi</title>
+  </head>
+  <body>
+    <noscript>You need to enable JavaScript to run this app.</noscript>
+    <div id="root"></div>
```
Comment (SergeOlabin) [inline public/index.html:33]: duplicate

**Review by SergeOlabin — CHANGES_REQUESTED** [review]

File: public/index.html (hunk 8 above)
Comment (darya42185) [reply public/index.html]: fixed

File: public/index.html [hunk 9]
```diff
@@ -24,6 +24,10 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <link rel="preconnect" href="https://fonts.gstatic.com/%22%3E"/> 
+    <title>Dokazovi</title>
+    <noscript>You need to enable JavaScript to run this app.</noscript>
+    <div id="root"></div>
```
Comment (SergeOlabin) [inline public/index.html:30]: duplicate

File: public/index.html [hunk 10]
```diff
@@ -24,6 +24,10 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <link rel="preconnect" href="https://fonts.gstatic.com/%22%3E"/> 
+    <title>Dokazovi</title>
+    <noscript>You need to enable JavaScript to run this app.</noscript>
```
Comment (SergeOlabin) [inline public/index.html:29]: pls move below the links

**Review by SergeOlabin — CHANGES_REQUESTED** [review]
pls resolve conflicts

**Review by dusty3ntity — APPROVED** [review]

**Pull request merged**